docs/source/userguide.rst

```diff
@@ -245,16 +156,50 @@ The steps for training a model with HPO are identical to the default :code:`Simp
     )
 
     # fit an imputer model with customized hyperparameters
-    imputer.fit_hpo(
-        train_df=df_train,
-        num_epochs=100,
-        patience=3,
-        learning_rate_candidates=[1e-3, 3e-4, 1e-4]
-    )
+    imputer.fit_hpo(train_df=df_train)
+
+Calling HPO like this will search through some basic and usually helpful hyperparameter choices.
+There are two ways for a more detailed search. Firstly, :code:`fit_hpo` offers additional arguments that can be inspected in the SimpleImputer_. For even more configurations and variation of hyperparameters for the various input column types, a dictionary with ranges can be passed to :code:`fit_hpo` as can be seen in the hpo-code_.
+Results for any HPO run can be accessed under :code:`imputer.hpo.results` and the model from any HPO run can then be loaded using :code:`imputer.load_hpo_model(idx)` passing the model index.
+
+
+Label Shift and Empirical Risk Minimization
```

Comment:
I think having this explanation of label-shift here could confuse readers. Maybe we should place this in a different section where we group advanced features.

Reply:
You mean place the entire documentation of this feature in a different section or only the explanation of what label-shift is?